notification_lib/novu_manager/generic_novu_manager.py

```diff
@@ -51,3 +51,49 @@ def get_generic_novu_templates(cls) -> list[dict[str, Any]]:
         logging.error(f"status code: {response.status_code}")
         logging.error(f"response : {response.json()}")
         raise NotificationException("Erreur lors de l'importation des templates")
+
+    @classmethod
+    def get_generic_template_by_id(cls, template_id: str) -> dict[str, Any]:
+        """Get generic template by providing his id."""
+        response = requests.get(
+            url=f"{cls.api_url_generic_novu}/v1/notification-templates/{template_id}",
+            headers={"Authorization": f"Bearer {cls.get_novu_jwt_bearer_token()}"},
+            timeout=5,
+        )
+
+        if response.status_code // 100 == 2:
+            result = response.json()["data"]
+            steps = []
+
+            def construct_filters(step_filter: dict[str, Any]) -> dict[str, Any]:
```

Comment:
why this function is inside get_generic_template_by_id function

Reply:
thought about it like a small function for formating, will change it to a static function